PH-1/app/ingestion/router.py

```
@@ -44,4 +46,31 @@ def validate_file_strict(file: UploadFile) -> None:
                 status_code=400,
                 detail=f"File too large. Maximum size for {filetype} files: {max_size_mb}MB"
-            )
\ No newline at end of file
+            )
+        
+async def extract_text_image(file: UploadFile) -> Optional[str]:
+    """Extract text from image using OCR"""
+    try:
+        file.file.seek(0)
+        image_bytes = file.file.read()
+        image = Image.open(io.BytesIO(image_bytes))
```

Ruff (0.11.9)

Undefined name `io`

(F821)